.travis.yml

```diff
@@ -1,26 +1,33 @@
 # See https://github.com/silverstripe-labs/silverstripe-travis-support for setup details
 
-language: php 
-php: 
- - 5.4
-
 sudo: false
 
+language: php
+
+php:
+  - 5.3
+  - 5.4
+  - 5.5
+  - 5.6
+
 env:
- - DB=MYSQL CORE_RELEASE=3.1
- - DB=MYSQL CORE_RELEASE=3
- - DB=PGSQL CORE_RELEASE=3.1
- - DB=PGSQL CORE_RELEASE=3
+  - DB=MYSQL CORE_RELEASE=3.2
 
 matrix:
   include:
-    - php: 5.3
+    - php: 5.6
+      env: DB=MYSQL CORE_RELEASE=3
+    - php: 5.6
       env: DB=MYSQL CORE_RELEASE=3.1
+    - php: 5.6
+      env: DB=MYSQL CORE_RELEASE=3.3
 
 before_script:
- - git clone git://github.com/silverstripe-labs/silverstripe-travis-support.git ~/travis-support
- - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss --require silverstripe/cms
- - cd ~/builds/ss
+  - composer self-update || true
+  - git clone git://github.com/silverstripe-labs/silverstripe-travis-support.git ~/travis-support
+  - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss --require silverstripe/cms
+  - cd ~/builds/ss
+  - composer install
```

Comment:
This line is actually not needed as `travis_setup.php` does it for us, but I guess it's no harm